kc-cloudtrust-module/kc-cloudtrust-common/src/main/java/io/cloudtrust/keycloak/RequiredActionUtils.java

```diff
@@ -0,0 +1,31 @@
+package io.cloudtrust.keycloak;
+
+import java.util.List;
+
+import javax.ws.rs.core.MultivaluedHashMap;
+import javax.ws.rs.core.MultivaluedMap;
+
+import org.jboss.resteasy.spi.HttpRequest;
+import org.keycloak.authentication.RequiredActionContext;
+
+public class RequiredActionUtils {
```

Comment:
As it looks like to be an exact copy of AuthenticatorUtils, should we not make a super class containing this code ?